src/Microsoft.Fast.Components.FluentUI/Components/Base/FluentInputBase.cs

```diff
@@ -240,7 +245,7 @@ protected FluentInputBase()
     {
         get
         {
-            string? fieldClass = EditContext?.FieldCssClass(FieldIdentifier);
+            string? fieldClass = !FieldBound ? string.Empty : EditContext?.FieldCssClass(FieldIdentifier);
```

Comment:
For CleanCode, it's better to have only "positive" condition:

```suggestion
            string? fieldClass = FieldBound ? EditContext?.FieldCssClass(FieldIdentifier) : null;
```

Reply:
Yes, better. Done!